src/shogun/kernel/GaussianKernel.cpp

```diff
@@ -16,9 +16,21 @@
 #include <shogun/kernel/GaussianKernel.h>
 #include <shogun/features/DotFeatures.h>
 #include <shogun/io/SGIO.h>
+#include <shogun/mathematics/Math.h>
 
 using namespace shogun;
 
+void CGaussianKernel::set_width(float64_t w)
+{
+	REQUIRE(w>0, "width (%f) must be positive\n",w);
+	m_log_width=CMath::log(w/2.0)/2.0;
```

Comment:
the `width` defined in `CGaussianKernel` is width=w*w*2.0, where `w` is the parameter used in GPML.